pkg/registry/apis/provisioning/jobs/migrate/worker.go

```diff
@@ -88,12 +89,21 @@ func (w *MigrationWorker) Process(ctx context.Context, repo repository.Repositor
 		buffered *gogit.GoGitRepo
 	)
 
+	progress.SetTotal(10) // will show a progress bar
 	if repo.Config().Spec.GitHub != nil {
-		progress.SetMessage("clone target")
+		progress.SetMessage("clone " + repo.Config().Spec.GitHub.URL)
+		reader, writer := io.Pipe()
+		go func() {
+			scanner := bufio.NewScanner(reader)
+			for scanner.Scan() {
+				progress.SetMessage(scanner.Text())
```

Comment:
you won't see anything with https://github.com/grafana/git-ui-sync-demo -- but with big repos this checkout can take a long time